webpack.config.js

```diff
@@ -47,6 +52,9 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
                 fileName: outputPath + '/manifest.json',
             }),
             new CleanObsoleteChunksPlugin(),
+            new webpack.DefinePlugin({
+                SULU_ADMIN_BUILD_VERSION: JSON.stringify(suluVersion),
```

Comment:
Why does the version need to be stringified? Shouldn't that already be a string? :thinking:

Reply:
the plugin does a direct text replacement, therefore the quotes must be included in the value. see the blue box in the docs: https://webpack.js.org/plugins/define-plugin/